crates/extensions/c8y_auth_proxy/src/server.rs

```diff
@@ -135,23 +146,127 @@ impl FromRef<AppState> for SharedTokenManager {
 }
 
 #[derive(Clone)]
-struct TargetHost(Arc<str>);
+struct TargetHost {
+    http: Arc<str>,
+    ws: Arc<str>,
+}
+
+fn axum_to_tungstenite(
+    message: axum::extract::ws::Message,
+) -> tokio_tungstenite::tungstenite::Message {
+    use axum::extract::ws::CloseFrame as AxumCf;
+    use axum::extract::ws::Message as AxumM;
+    use tokio_tungstenite::tungstenite::protocol::frame::CloseFrame as TungCf;
```

Comment:
It would be good to find another prefix as `Tung` confuses spell checking.

Reply:
I've now renamed this to `InCf`/`OutCf` like the other function that does essentially the same thing.